### DISCUSSION TITLE
Using digitalocean spaces cdn?

### DISCUSSION THREAD

**Discussion by virus2016** [directus/directus#4740] [Archive]
Hi there,

I have successfully added my digitalocean spaces account (Amazon s3 equivalint) however I notice that the direct access to assets using the non CDN endpoint is slow. They give you the following:

Origin: https://[BUCKET_NAME].fra1.digitaloceanspaces.com
Edge (CDN): https://[BUCKET_NAME].fra1.cdn.digitaloceanspaces.com

Is there an option to add the CDN as and assets host so that we can benefit from the speed increases?

**Comment by rijkvanzanten** [comment]
Directus doesn't have control over the endpoints on DigitalOcean Spaces, so there's no way for Directus to have an assets endpoint on top of the DO Spaces URL.

That being said, you can put a CDN in front of Directus and enjoy the best of both worlds.
